tardis/plasma/properties/ion_population.py

```diff
@@ -136,36 +137,31 @@ class RadiationFieldCorrection():
     This function adds a field 'delta' to the phi table given to the function
 
     """
-
-    def calculate(self, w, departure_coefficient=None, chi_0_species=(20, 2)):
-        #factor delta ML 1993
-        if departure_coefficient is None:
-            departure_coefficient = 1. / self.ws
-
-        ionization_data = self.atom_data.ionization_data
-
+    @staticmethod
+    def calculate(w, ionization_data, beta_rad, t_electron, t_rad,
+        beta_electron, levels):
+        # factor delta ML 1993
+        departure_coefficient = 1. / w
+        chi_0_species=(20, 2)
```

Comment:
@wkerzendorf In the old plasma, `departure_coefficient` and `chi_0_species` were both arguments for this function, with these default values. But there didn't seem to be anywhere where they could actually be altered to be anything other than these values. Should I make them their own `ProcessingPlasmaProperty` objects, or should I just leave their values constant?